lib/algo/ParametricDMDc.h

```diff
@@ -0,0 +1,192 @@
+/******************************************************************************
+ *
+ * Copyright (c) 2013-2023, Lawrence Livermore National Security, LLC
```

Comment:
2024

Reply:
Updated the copyright years. Also updated the documentation at the start of `parametricDMDc` because I never changed it from what was in `parametricDMD`